### PR TITLE
[SF7] Add Symfony 7 support for some components

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
|-----------------|-----
| Branch?         | symfony-7
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

After
image

### CODE REVIEW

File: src/Sylius/Bundle/AddressingBundle/composer.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
             "dev-main": "2.0-dev"
         },
         "symfony": {
-            "require": "6.3.*"
+            "require": "7.0.*"
```
Comment (diimpp) [inline src/Sylius/Bundle/AddressingBundle/composer.json:63]: I assume this PR targets sylius 2.0, so it's alright to specify `require 7.0.*`

File: src/Sylius/Bundle/CustomerBundle/composer.json [hunk 2]
```diff
@@ -62,6 +62,9 @@
     "extra": {
         "branch-alias": {
             "dev-main": "2.0-dev"
+        },
+        "symfony": {
+            "require": "7.0.*"
```
Comment (diimpp) [inline src/Sylius/Bundle/CustomerBundle/composer.json:67]: Do you need to specify this on every composer? At least some of them missing it, e.g. Channel, Attribute, Api, Locale, Order, Payment bundles

**Review by diimpp — APPROVED** [review]

File: src/Sylius/Bundle/CustomerBundle/composer.json (hunk 2 above)
Comment (loic425) [reply src/Sylius/Bundle/CustomerBundle/composer.json:67]: That's not mandatory, cause CI does this, but to test it locally, this is necessary to do it to be sure you only install Symfony 7 packages.

Comment (loic425) [reply src/Sylius/Bundle/CustomerBundle/composer.json:67]: Right, I'll do that on missing packages.

**Pull request merged**